src/share/classes/com/sun/javatest/regtest/config/RegressionParameters.java

```diff
@@ -1288,7 +1288,7 @@ public String getTestThreadFactory() {
     private String testThreadFactory;
 
     public void setTestThreadFactoryPath(String testThreadFactoryPath) {
-        this.testThreadFactoryPath = testThreadFactoryPath;
+        this.testThreadFactoryPath = new File(testThreadFactoryPath).getAbsolutePath();
```

Comment:
Let's use NIO `Path` here.

```suggestion
        this.testThreadFactoryPath = Path.of(testThreadFactoryPath).toAbsolutePath().toString();
```